File: 기초 사고력문제/백준_2577_숫자의 갯수/숫자의 갯수.java
import java.util.*;

public class Main{
    public static void main(String[] args){
        Scanner s = new Scanner(System.in);
        int a = s.nextInt();
        int b = s.nextInt();
        int c = s.nextInt();
        int mul = a*b*c;
        int[] arr = new int[10];

        while(mul >= 1){
            int i = mul % 10;
            arr[i]++;
            mul /= 10;
        }
        for(int i = 0; i< arr.length; i++) System.out.println(arr[i]);
    }
}

File: 기초 사고력문제/백준_2577_숫자의 갯수/숫자의 갯수.cs
using System;
using static System.Console;

namespace cs_algo
{
    class Program
    {
        public static void  Main(string[] args)
        {
            int a = int.Parse(ReadLine());
            int b = int.Parse(ReadLine());
            int c = int.Parse(ReadLine());
            int mul = a * b * c;
            int[] arr = new int[10];

            while (mul >= 1)
            {
                int i = mul % 10;
                arr[i]++;
                mul /= 10;
            }

            for (int i = 0; i < arr.Length; i++) WriteLine(arr[i]);
        }
    }
}